hudi-utilities/src/test/java/org/apache/hudi/utilities/TestHoodieDeltaStreamer.java

```diff
@@ -653,7 +659,7 @@ private void prepareParquetDFSSource(boolean useSchemaProvider, boolean hasTrans
     if (useSchemaProvider) {
       parquetProps.setProperty("hoodie.deltastreamer.schemaprovider.source.schema.file", dfsBasePath + "/source.avsc");
       if (hasTransformer) {
-        parquetProps.setProperty("hoodie.deltastreamer.schemaprovider.source.schema.file", dfsBasePath + "/target.avsc");
+        parquetProps.setProperty("hoodie.deltastreamer.schemaprovider.target.schema.file", dfsBasePath + "/target.avsc");
```

Comment:
did you have to fix any of the test because of this change? If not, I am afraid that our tests did not catch this.

Reply:
I don't remember fixing unit tests.  Given that this is optional so it is possible that the data written may be different from the schema designated.  However, I think the integration tests should be able to catch any issue due to schema mismatch.